train.py

```diff
@@ -170,6 +170,18 @@ def validate_directories(args):
     }
 
 
+def check_sample_size(dilations, sample_size):
+    '''Check that the sample size is greater than the receptive field size and
```

Comment:
The first sentence should also fit in a line, but this is also not enforced elsewhere. For instance, you don't need to talk about the assertion yet. Or, even better, name it `assert_sample_size` so that what it does is obvious.

You should then add a section with the Args, and a Raises section where you specify which exception is raised.